dlt/destinations/job_client_impl.py

```diff
@@ -197,8 +197,11 @@ def drop_tables(self, *tables: str, delete_schema: bool = True) -> None:
             delete_schema: If True, also delete all versions of the current schema from storage
         """
         with self.maybe_ddl_transaction():
-            self.sql_client.drop_tables(*tables)
-            if delete_schema:
+            # only run drop command on existing tables
+            existing_tables = self.get_known_existing_storage_tables()
+            tables_to_drop = set(existing_tables).intersection(set(tables))
```

Comment:
this will only issue drop commands for tables that exist in the destination AND are known by dlt. This is a light change from before where any table could be dropped. Is this ok?